### DISCUSSION TITLE
Connecting to two LDAP servers for two different domains

### DISCUSSION THREAD

**Discussion by fenchi** [authelia/authelia#6289] [Q&A] [answered]
### Description

Hello,

Is it possible to configure Authelia to two LDAP servers for two different domains?
When I look at the documentation, I see that you can only connect to one LDAP.

I have an idea but I'm not sure if it would work.
It would be to launch two Authelia instances, one for each LDAP. The two instances share data on a Postgres server and cache on a Redis server.
Since the data is shared between the two instances, an authentication made by one persists for the other.
I think this will cause a problem for authentication, so I suggest that my reverse proxy redirect to an HTML page where I ask the user from which LDAP domain they want to authenticate.

Can this idea work?

Best regards,

### Use Case

I am using Authelia for 2FA on a set of web servers and as an OIDC provider.

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment, marked as answer]
As long as the usernames in both LDAP environments are the same the credentials will be shared. As far as the separate domains.. depends if the domains are valid to share cookies per web standards. i.e. if you have example1.com and example2.com then the user will have to login again.

If you mean app1.example.com and app2.example.com and you set the cookie domain to example.com then it should work.
